lib/active_merchant/billing/gateways/global_collect.rb

```diff
@@ -277,9 +277,13 @@ def uri(action, authorization)
         end
       end
 
-      def commit(action, post, authorization = nil)
+      def idempotency_key(options)
```

Comment:
Love this method. Do you think it would be better named `idempotency_key_for_signature` to distinguish it from the actual header that gets added on line 320? Just a suggestion, I defer to your judgement on it.